ci/create-aro-cluster.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+echo "Installing Azure CLI"
+
+curl -L https://aka.ms/InstallAzureCli | bash
```

Comment:
I take it that there isn't a better-trusted way to acquire this? There's a whole beautiful thread on memo-list about this very thing and how scary it is.

Reply:
There are, depending on the distro. But this future proofs any base-image changes and it's supported via the Azure docs so I'm comfortable with it.